lib/rucio/core/subscription.py

```diff
@@ -211,8 +213,8 @@ def list_subscriptions(name=None, account=None, state=None, session=None):
         if state:
             query = query.filter_by(state=state)
     except IntegrityError as error:
-        print(error)
-        raise
+        logger(logging.ERROR, str(error))
+        raise Exception
```

Comment:
this doesn't make much sense to me. Why replace the re-raise with a generic Exception without any information?